rtc/configuration.md

```diff
@@ -136,143 +136,198 @@ tinymce.init({
 })
 ```
 
-## Optional configuration
+## Recommended configuration
 
-### `rtc_snapshot`
+For the best user experience, {{site.companyname}} recommends including these configuration settings:
 
-Real-time collaboration integrations regularly store the content, eliminating the need for a save button. The {{site.productname}} RTC plugin provides a version number to assist with storing the regular content snapshots. These snapshots are not stored by the {{site.cloudname}} and must be handled by the integrator.
+* [`rtc_server_disconnected`](#rtc_server_disconnected)
+* [`rtc_user_details_provider`](#rtc_user_details_provider)
 
-For any given document ID, the server guarantees the version number will only increase. It can be safely used for conflict resolution. For each document ID and version combination the snapshot content is guaranteed to be identical.
+### `rtc_server_disconnected`
 
-The snapshot callback will be executed at regular intervals with access to the serialized editor content. The content is retrieved through a `getContent` function to reduce CPU load if the callback decides to not use the editor content.
+If the RTC session fails to connect, or is disconnected due to an error, the user will be blocked from editing (using [setProgressState]({{site.baseurl}}/api/tinymce/tinymce.editor/#setprogressstate)) along with an error notification:
+
+![RTC disconnected error example]({{site.baseurl}}/images/rtc-error-example.png "RTC disconnected error example")
```

Comment:
I was able to slim this image down by 55% with ImageOptim. Probably worth doing

![Screen Shot 2021-07-07 at 1 37 18 pm](https://user-images.githubusercontent.com/2288242/124696448-746e3e80-df28-11eb-94db-a5fd35823aa6.png)

Reply:
I always forget to do that - but to be fair I wasn't even sure there should _be_ a screenshot. I was hoping for a doc or PM review before this progresses.

Reply:
I ran tinypng on the whole image folder - it shrinks the RTC error png a lot more than ImageOptim (final size 9404 bytes) - and saved 2,534,787 bytes. I'll make a separate PR for that, though.